### ISSUE TITLE
Game crashes after placing a statue with invalid name

### ISSUE THREAD

**Issue by pokemoneuro** [ShyNieke/StatuesMod#27]
Hi, I just encountered an issue with your mod. My friend just placed a renamed statue with an apparently invalid name and our games crashed with the message that the game encountered a problem while rendering an entity in the game. 
Maybe this is something to address

**Comment by Mrbysco** [comment]
Could you be more specific? Could you provide the crash log?

**Comment by pokemoneuro** [comment]
We got statues from dying in PvP. He renamed one and he misspelled a name. He placed it into the world and we crashed. Then the world file corrupted. It said that there was a fault during the rendering of the entity. As soon as he placed the thing down, our clients froze and it crashed
[2019-06-20-2.log](https://github.com/svennieke/StatuesMod/files/3311977/2019-06-20-2.log)
I think the attached file is of the crash on the server. 
[crash-2019-06-20_15.14.24-client.txt](https://github.com/svennieke/StatuesMod/files/3311981/crash-2019-06-20_15.14.24-client.txt)
The second file is the crash report from my client.
